raiden/messages.py

```diff
@@ -995,6 +1116,40 @@ def from_event(event):
             fee,
         )
 
+    def to_dict(self):
+        return {
+            'identifier': self.identifier,
+            'nonce': self.nonce,
+            'token': self.token.hex(),
+            'channel': self.channel.hex(),
+            'transferred_amount': self.transferred_amount,
+            'recipient': self.recipient.hex(),
+            'lock': self.lock.to_dict(),
+            'locksroot': self.locksroot.hex(),
+            'target': self.target.hex(),
+            'initiator': self.initiator.hex(),
+            'fee': self.fee,
+            'signature': self.signature.hex(),
+        }
+
+    @staticmethod
+    def form_dict(data):
```

Comment:
also here

Reply:
I left some behind to check you were paying attention